crates/chronicle/src/bootstrap/cli.rs

```diff
@@ -1056,6 +1056,33 @@ impl SubCommand for CliModel {
                         .value_parser(["data", "graphql"])
                         .default_values(&["data", "graphql"])
                         .help("which API endpoints to offer")
+                    )
+                    .arg(
+                        Arg::new("load-opa-from-url")
```

Comment:
```suggestion
                        Arg::new("opa-policy-address")
```
Just a suggestion, not a blocker. (Thinking that the prefix fits the other OPA policy stuff and the suffix fits other URL stuff among our options.) Curious about other ideas though given that "policy address" has a non-tech meaning!

People might appreciate an environment variable for these too.